examples/cloudml-collaborative-filtering/preprocessing/preprocess.py

```diff
@@ -44,6 +44,17 @@ def ReadBQ(p, query):
       query=query, use_standard_sql=True))
 
 
+def _normalize_user_tags(features):
+  """Fills in missing user tags features and normalizes the tag vectors."""
```

Comment:
This docstring should describe the type/structure of the expected input.

nit: "fills in" implies that you are mutating the input value in-place but this function actually returns the modified value.